### PR TITLE
LPS-190173 the system FF stuff

### PR DESCRIPTION
**Pull request by thiago-buarqque**

### CODE REVIEW

**Comment by thiago-buarqque** [comment]
ci:test:relevant

**Comment by thiago-buarqque** [comment]
ci:relevant:stop

**Comment by thiago-buarqque** [comment]
ci:test:relevant

File: modules/apps/feature-flag/feature-flag-web/src/main/java/com/liferay/feature/flag/web/internal/configuration/admin/category/FeatureFlagConfigurationCategory.java [hunk 1]
```diff
@@ -54,7 +55,15 @@ protected void activate(BundleContext bundleContext) {
...
-					new FeatureFlagConfigurationScreen(
+					new CompanyFeatureFlagConfigurationScreen(
+						_featureFlagManager, featureFlagType,
+						_featureFlagsDisplayContextFactory, _servletContext),
+					new HashMapDictionary<>()));
+
+			_serviceRegistrations.add(
+				bundleContext.registerService(
+					ConfigurationScreen.class,
+					new SystemFeatureFlagConfigurationScreen(
```
Comment (drewbrokke) [inline modules/apps/feature-flag/feature-flag-web/src/main/java/com/liferay/feature/flag/web/internal/configuration/admin/category/FeatureFlagConfigurationCategory.java:66]: No need for separate types on this one. Just pass in the scope as another parameter.

**Comment by thiago-buarqque** [comment]
Just sent the PR to the PEDS repo see [#248](https://github.com/liferay-peds/liferay-portal/pull/248).

**Pull request closed without merging**